iree/compiler/Dialect/VM/Target/C/vm_c_funcs.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Google LLC
```

Comment:
I think we should move this into iree/vm/; that way all code that is being cross-compiled for targets is not dependent on anything under iree/compiler/ (and you could, in theory, not have iree/compiler/ set up in your build system at all after the code has been generated)

Reply:
Thanks for the suggestion! I shuffled the file and the build passes locally. Thus, I would appreciate if you could merge the PR if the build pipeline I just triggered passes.